teleop/htm/templates/mobile_controller_ui.html

```diff
@@ -1,17 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link rel="stylesheet" type="text/css" href="../static/CSS/mobileController.css">
   <link rel="shortcut icon" href="#" />
   <title>Mobile controller</title>
+  <style>
+    /* CSS styles for layout */
+    .mobile_view_container {
+      display: flex;
+      flex-direction: column;
+      position: relative;
+    }
+
+    .input-container {
+      position: absolute;
+      display: flex;
+      flex-direction: column;
+      align-items: left;
+      top: 20px;
+    }
+
+    .input-group {
+      font-size: 14px;
+      display: flex;
+      align-items: center;
+      margin-left: 10px;
+    }
+
+    .input-group label {
+      width: 110px;
+    }
+
+    .input-box-container {
+      display: flex;
+      align-items: center;
+    }
+
+    .input-box {
+      width: 50px;
+    }
+
+    .input-box-text {
+      margin-left: 10px;
+    }
+
+    .button-container {
+      margin-top: 10px;
+      text-align: right;
+    }
+
+    .hidden {
+      display: none;
+    }
+
+    /* Add this part to make the screen "stable".
+    Without this, the screen can be moved to the right or left,
+    making objects on the canvas appear inconsistent. */
+    html, body {
+      padding: 0;
+      margin: 0;
+    }
+  </style>
 </head>
 
 <body>
-  <script src="../static/external/pixi-v7.3.0-rc.2.min.js"></script>
-  <script type="module" src="../static/JS/mobileController/mobileController_a_app.js"></script>
-</body>
+  <div class="mobile_view_container">
+    <!-- Input container -->
+    <div class="input-container">
```

Comment:
Please be more specific with naming tags. There might be another `input-container` but there won't be more of `mobile-controller-motor-settings-container`